packages/interaction-kit/src/structures/markdown.ts

```diff
@@ -0,0 +1,82 @@
+import { URL } from "url";
+import { Mentionable } from "../interfaces";
+
+export function user(user: Mentionable) {
+	return `<@${user.id}>`;
+}
+
+export function nickname(user: Mentionable) {
+	return `<@!${user.id}>`;
+}
+
+export function channel(channel: Mentionable) {
+	return `<#${channel.id}>`;
+}
+
+export function role(role: Mentionable) {
+	return `<@&${role.id}>`;
+}
+
+export function emoji(emoji: { name: string } & Mentionable, animated = false) {
+	if (animated) {
+		return `<a:${emoji.name}:${emoji.id}>`;
+	}
+
+	return `<:${emoji.name}:${emoji.id}>`;
+}
+
+export enum TimestampStyle {
+	SHORT_TIME = "t",
+	LONG_TIME = "T",
+	SHORT_DATE = "d",
+	LONG_DATE = "D",
+	SHORT_DATETIME = "f",
+	LONG_DATETIME = "F",
+	RELATIVE = "R",
+}
+
+export function time(timestamp: number, type = TimestampStyle.SHORT_DATETIME) {
+	return `<t:${timestamp}:${type};`;
```

Comment:
There is a `>` missing at the end and the `;` doesn't belong here.

Reply:
Good catch, thanks!